.gitignore

```diff
@@ -2,6 +2,7 @@ node_modules
 
 *.log
 *.log.*
+.idea/
```

Comment:
Not really sure what this one is for 🤔

Reply:
I opened the project in my IDE (IntelliJ) which always creates this folder to store project-related settings. I can remove this if you don't feel the need. ;)

Reply:
You could just add the same thing in the other file?